app/views/examples/vehicle-check-answers.html

```diff
@@ -0,0 +1,88 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  Check your answers
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+  
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">
+        Check your answers before sending your application
+      </h1>
+    </div>
+  </div>
+
+  <table class="check-your-answers">
+
+    <thead>
+      <tr>
+        <th colspan="2">
+          <h2 class="heading-medium">
+            Vehicle details
+          </h2>
+        </th>
+        <th>
+        </th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <tr>
+        <td>
+          Registration number
+        </td>
+        <td>
+          {{vehicleRegistration}}
+        </td>
+        <td class="change-answer">
+          <a href="/examples/pass-data">
+            Change <span class="visuallyhidden">registration number</span>
+          </a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Vehicle type
+        </td>
+        <td>
+          {{vehicleType}}
+        </td>
+        <td class="change-answer">
+          <a href="/examples/vehicle-type">
+            Change <span class="visuallyhidden">vehicle type</span>
+          </a>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          Vehicle features
+        </td>
+        <td>
+          {{vehicleFeature}}
+        </td>
+        <td class="change-answer">
+          <a href="/examples/vehicle-type">
+            Change <span class="visuallyhidden">vehicle features</span>
+          </a>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+
+  <h2 class="heading-medium">Now send your application</h2>
+
+  <p class="text">
+    By submitting this notification you are confirming that, to the best of your knowledge, the details you are providing are correct.
+  </p>
+
+  <div class="form-group">
+    <a href="confirmation-page" class="button">Accept and send application</a>
+  </div>
+
+</main>
```

Comment:
It feels like this page could have some kind of 'summary' paragraph describing how you then use the data - eg an example of the markup to use in the page.

Would also be good to have a link back to the main examples - so users have a return path.